docs/src/pages/index.mdx

```diff
@@ -132,7 +132,7 @@ _.flow(
   _.filter((x) => x.gender === "f"),
   //              ^? 😔 'x' is of type 'unknown'
   _.groupBy((x) => x.age),
-  //                 ^? 😔 Property 'age' does not exist on type 'P'
+  //               ^? 😔 'x' is of type 'unknown'
```

Comment:
I noticed this was slightly off from what I observed. Probably doesn't make a big difference, but while I was in there I thought I would tweak it. Totally fine if you want me to revert or even dump this whole line if you don't think it adds much (since it is the same error as above).

TS Playground for reference if you want to see the errors: https://tinyurl.com/569s526w